services/core-web/src/styles/components/Callout.scss

```diff
@@ -0,0 +1,45 @@
+@mixin base-callout {
+    padding: 25px;
+    margin: 16px 0;
+    background-color: #f2f2f2;
+    font-size: 20px;
+}
+
+.bcgov-callout {
+    &--success {
+        @include base-callout;
+        border-left: 10px solid #5cb85c;
+    }
+
+    &--warning {
+        @include base-callout;
+        border-left: 10px solid #f0ad4e;
+    }
+
+    &--danger {
+        @include base-callout;
+        border-left: 10px solid #d9534f;
+    }
+
+    &--default {
+        @include base-callout;
+        border-left: 10px solid #d9534f;
+    }
+}
+
+.bcgov-callout--success>p,
+.bcgov-callout--warning>p,
+.bcgov-callout--danger>p {
+    margin-bottom: 0;
+}
+
+.bcgov-callout h1,
+h2,
```

Comment:
I see you're bringing this file over to CORE!

An issue that I noticed with this on Minespace: this selector doesn't work as intended. They meant for every `h1`, `h2`, etc, _within_ `.bcgov-callout` to be selected, but what actually happened was they selected the `h1` within bcgov-callout and also _every single other_ `h2`-`h4`! I changed the selector, and then made some other changes that sonarcloud or eslint wanted, in the PR that I put up today.

Reply:
Thanks for the heads up. I adjusted this to be aligned with the change you made in your pr.